src/views/Layout/Layout.js

```diff
@@ -1,16 +1,20 @@
+import React from "react";
```

Comment:
```suggestion
```

Unless used explicitly in the file, react should not be imported.  Some linters are outdated